deploy/examples/operator.yaml

```diff
@@ -35,6 +35,8 @@ data:
   ROOK_CSI_ENABLE_RBD: "true"
   # Enable the CSI NFS driver. To start another version of the CSI driver, see image properties below.
   ROOK_CSI_ENABLE_NFS: "false"
+  # Enable the CSI driver.
+  ROOK_CSI_DISABLE_CSI_DRIVER: "false"
```

Comment:
`CSI_DISABLE_DRIVER` or `ROOK_CSI_DISABLE_DRIVER` might be the better name and comment.
1. Update the comment to says set this to diable csi driver
2. add the same to operator-openshift.yaml and helm charts

Reply:
Updated

Reply:
How about `DISABLE_CSI_DRIVER`? Currently used `ROOK_CSI_DISABLE_DRIVER`

Reply:
trying to keep `ROOK` or `CSI` as starting key. `ROOK_CSI_DISABLE_DRIVER` is also fine